### DISCUSSION TITLE
v2.2.0

### DISCUSSION THREAD

**Discussion by MidKnightXI** [MidKnightXI/opgg-ads-remover#26] [Announcements]
# What's changed
- Reduced the binary file by around 2/3 its original size
> I configured Tauri to reduce binary size and removed the "all-feature" flag and removed unused fonts from the CSS.

<hr /><em>This discussion was created from the release <a href='https://github.com/MidKnightXI/opgg-ads-remover/releases/tag/v2.2.0'>v2.2.0</a>.</em>

**Comment by frankieyim** [comment]
hi pal...it didnt work in latest version for OP.GG 1.128 ...please kindly to update..thanks

**Reply by MidKnightXI** [reply]
I'll check that when I've time, please open an issue with the requested informations 😄
